kcidb/monitor/output.py

```diff
@@ -174,18 +177,22 @@ def render(self):
             self.obj.get_type().name: self.obj,
         }
         subject = TEMPLATE_ENV.from_string(self.message.subject).render(ctx)
-        subject_extra_bytes = \
-            len(subject.encode()) - NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN
-        assert subject_extra_bytes <= 0, \
-            f"Subject is {subject_extra_bytes} bytes too long"
+        subject_extra_characters = \
+            len(subject) - NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN
+        if subject_extra_characters > 0:
+            subject = subject[:NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN]
```

Comment:
It would be good to leave an indication to the notification recipient, that the subject was truncated. E.g. by adding some sort of character on the end, say "✂️". However, if we add that character after truncating on the limit, the subject will go over the limit. So we need to truncate to the limit, minus one, and then add the character. Could you do that? And same for the body?